src/components/section/Section.tsx

```diff
@@ -24,7 +25,12 @@ export function Section(props: IProps) {
       <SectionHeading text={sectionTexts} fallback={props.section.beskrivendeId} />
```

Comment:
Skal vi vise seksjonens beskrivelse og eventuelle hjelpetekst i netto-versjonen av PDFen? Jeg har en antakelse om at vi ikke skal det, og da må `SectionHeading` også få sendt med parameteret. Hadde det vært en idé å kalle parameteret `showAllTexts` i stedet for `showAllFaktumTexts`?

Reply:
Det stemmer 👌  I fix